### PR TITLE
Books collection

### PR DESCRIPTION
**Pull request by mugishasam123**
- [ ] add index.html
- [ ] add styles for index.html
- [ ] add javascript to enable adding and removing book
- [ ] linters check 
- [ ] removed uneccessary files.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (Dipeshtwis) [inline .vscode/settings.json]: - [x] I would recommend putting the unnecessary files or folders into the `.gitignore` file and avoid pushing on Github.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,41 @@
...
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
+To get the content of this project locally you need to run this command in your terminal:
+- git clone your https://github.com/matovu-farid/My-Portfolio.git
+- cd project My-Portfolio
```
Comment (Dipeshtwis) [inline README.md:21]: - [x] I would recommend mentioning all the steps to run the project locally. Here next step is `open index.html` file to see the app running.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mugishasam123 and @matovu-farid, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Add and remove functionality is working.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 3]
```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
+package.json
```
Comment (D4nielJ) [inline .gitignore]: - [x] Please do not add package-lock.json or package.json to `.gitignore`. These two are very important for those who might need to do any package installation to run the project locally (to run linters for example). Instead, add the files the last reviewer commented on.

File: .vscode/settings.json (hunk 1 above)
Comment (D4nielJ) [reply .vscode/settings.json]: Like this `.vscode/settings` file which local configuration from your IDE.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @matovu-farid and @mugishasam123, 

Great work on making the changes requested by the previous reviewer(s) 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @mugishasam123 , 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**